index.bs

```diff
@@ -551,6 +559,59 @@ It must act as follows:
   </pre>
 </div>
 
+<h4 id="stream-method-algo">
+The stream method</h4>
```

Comment:
I guess this is technically up to you, but I'd prefer not wrapping after an opening tag or before a closing tag as it introduces whitespace where there should be none.

Reply:
Also, if these are to have headings we should use `<code>stream()</code>` or some such as that's the typical way of writing down a method name.

Reply:
I was mostly trying to be consistent with the rest of the spec, but yeah agree that this should be done differently. I've cleaned up the rest of the spec and rebased this on top of that.